src/utils/helpers.js

```diff
@@ -0,0 +1,61 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+
+const salt = bcrypt.genSaltSync(10);
```

Comment:
Move the salt to encrypting method and use const salt =await bcrypt.genSalt(10); instead